libraries/botbuilder-dialogs-declarative/package.json

```diff
@@ -20,29 +20,21 @@
   "main": "./lib/index.js",
   "typings": "./lib/index.d.ts",
   "dependencies": {
-    "@microsoft/recognizers-text-choice": "1.1.4",
-    "@microsoft/recognizers-text-date-time": "1.1.4",
-    "@microsoft/recognizers-text-number": "1.1.4",
-    "@microsoft/recognizers-text-suite": "1.1.4",
-    "@types/jsonpath": "^0.2.0",
-    "@types/node": "^10.17.27",
-    "botbuilder-core": "4.1.6",
-    "botbuilder-dialogs": "4.1.6",
-    "botframework-connector": "4.1.6",
-    "botframework-schema": "4.1.6",
-    "jsonpath": "^1.0.0",
-    "node-watch": "^0.6.3"
+    "chokidar": "^3.4.0",
+    "jsonpath": "^1.0.0"
   },
   "devDependencies": {
     "@types/mocha": "^5.2.7",
+    "@types/node": "^10.17.27",
+    "@types/jsonpath": "^0.2.0",
     "codelyzer": "^4.1.0",
     "mocha": "^6.2.3",
     "nyc": "^15.1.0",
     "source-map-support": "^0.5.3",
     "ts-node": "^4.1.0"
   },
   "scripts": {
-    "test": "tsc && nyc mocha tests/",
+    "test": "tsc && nyc mocha tests/ --exit",
```

Comment:
Intentional?

Reply:
Yes, we added some tests to test if resource explorer can monitor folder changes continuously, which may prevent mocha tests exiting as usual. Any better suggestions?